cli/cli.ts

```diff
@@ -1166,6 +1165,10 @@ function uploadCoreAsync(opts: UploadOptions) {
         })
 }
 
+function saveLocalPxTarget(cfg: pxt.TargetBundle) {
+    fs.writeFileSync("pxtarget-temp.json", JSON.stringify(cfg, null, 4));
```

Comment:
Woops this "-temp" is a leftover from my testing. I actually meant to overwrite the existing pxtarget.json.